### PR TITLE
PP9-11644 - Shares: content item with long name is not visible in Overview window in Share link

### PR DESCRIPTION
**Pull request by tc4rdpp**
- panel component: fixed long title, rearranged css, added tooltip for title
 - content details dialog: fixed long title causing preview not to show
 - share settings panel: fixed long list item values, added tooltip for list item values

### CODE REVIEW

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/content-details-dialog/content-details-dialog.component.scss [hunk 1]
```diff
@@ -82,6 +82,7 @@
       .content-info {
         float: left;
         flex: 30;
+        max-width: 30%;
```
[the remark this answers is not in the document]
Comment (tc4rdpp) [reply src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/content-details-dialog/content-details-dialog.component.scss:85]: Done

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/share-settings-panel/share-settings-panel.component.html [hunk 2]
```diff
@@ -7,7 +7,7 @@
       <li>{{ 'ShareSettingsPanel.ModificationDate' | pptranslate }}</li>
     </ul>
     <ul class="settings-info">
-      <li>{{ subject }}</li>
+      <li [matTooltip]="subject">{{ subject }}</li>
```
[the remark this answers is not in the document]
Comment (tc4rdpp) [reply src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/share-settings-panel/share-settings-panel.component.html]: Done

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/shared-module/components/panel/panel.component.html [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <div class="panel-container">
   <div class="panel-header">
-    <h3 *ngIf="title">{{ title }}</h3>
+    <h3 *ngIf="title" [matTooltip]="title">{{ title }}</h3>
```
[the remark this answers is not in the document]
Comment (tc4rdpp) [reply src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/shared-module/components/panel/panel.component.html]: Done

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/shared-module/components/panel/panel.component.scss [hunk 4]
```diff
@@ -6,24 +6,22 @@ pp-panel,
 
   .panel-header {
     position: relative;
```
[the remark this answers is not in the document]
Comment (tc4rdpp) [reply src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/shared-module/components/panel/panel.component.scss:8]: Done

**Review by neotek — APPROVED** [review]
LGTM

**Pull request merged**